test/e2e/network/conntrack.go

```diff
@@ -328,25 +297,3 @@ func dumpConntrack(cs clientset.Interface) {
 		}
 	}
 }
-
-func dumpIptables(cs clientset.Interface) {
```

Comment:
Can we revert only the iptables part?

This is problematic because we log it for all nodes (5k), but logging conntrack should be fine as we log that for two nodes...

@aojea - FYI

Reply:
yeah sure, , sorry,

Reply:
@wojtek-t, I'm not sure if I understand your comment: I think both dumpIptables and dumpConntrack iterates over all nodes.

Reply:
Sorry - I think this has changed in the meantime, I think we were dumping only particular nodes in the past.